README.md

```diff
@@ -1523,6 +1523,7 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 **[`^        back to top        ^`](#)**
 
+- [Hopper](https://github.com/BRA1L0R/hopper-rs) - Run multiple servers under the same IP with this fast, configurable, lightweight reverse proxy for Minecraft written in Rust to achieve maximum efficiency and performance. Deploy it using the available docker image or by downloading the release. `MIT` `Rust/Docker`
```

Comment:
> Hopper is a lightweight reverse proxy for minecraft

Since it's specific to a game I think it should be mved under https://github.com/awesome-selfhosted/awesome-selfhosted#games---administrative-utilities--control-panels

Reply:
@BRA1L0R can you please move Hopper under `Games - Administrative Utilities & Control Panels` ?